java/src/jmri/jmrit/operations/trains/JmritOperationsTrainsBundle_da.properties

```diff
@@ -549,7 +549,7 @@ buildErrorReqDest       = Train ({0}) requires {1}, can''t set out {1} at destin
 buildErrorNoMoves       = No moves available at location ({0}) id {1} can''t pick up {2}
 buildErrorRequirements  = Train ({0}) requires {1}, can''t meet requirements at departure ({2}) and/or termination ({3})
 buildErrorRouteLoc      = Route ({0}) missing location ({1})
-buildErrorRsNoLoc   = ERROR Exclude ({0}) at location ({1}) no track assignment
+buildWarningRsNoTrack   = WARNING! Exclude ({0}) at location ({1}) no track assignment
```

Comment:
If the strings are not translated, it's better to remove them from non English property files, than having the English text in the non English property files. It makes it easier for translators to find the strings that needs to be translated.
There was a discussion about this on the jmri developers list a while ago.